modules/ossm-deploy-application-workloads-per-user-group.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+// install/ossm-using-multiple-control-planes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-per-user-group_{context}"]
+= Deploy application workloads per user group
+:context: ossm-deploy-application-workloads-per-user-group
+
+This procedure demonstrates how to deploy application workloads in separate namespaces based on user groups.
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Label the namespace with the respective control plane:
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 usergroup=usergroup-1 istio.io/rev=usergroup-1
+----
+
+. Deploy the `sleep` and `httpbin` applications:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/sleep/sleep.yaml -n app-ns-1
```

Comment:
@FilipB Similar issue here with referencing upstream artifacts. If we want to use these, they should be from our downstream "openshift-service-mesh" fork. 

For httpbin, we should also reference a release branch rather than master branch in case it gets removed in a future release.

Would the following work instead?
https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml
https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml

Reply:
IIRC the version of httpbin from release-1.23 was not working so it was necessary to use the version from master. It seems it's [fixed](https://github.com/istio/istio/commit/6649dd8ed6b6cb58a3e04598fa72a25cf539d58b) now even in release-1.23 so it's safe to use it.
Should I sync changes from here to openshift-service-mesh/sail-operator repo?